### ISSUE TITLE
Got error indicators in many internal files

### ISSUE THREAD

**Issue by Lauriichan** [Prunoideae/ProbeJS#57]
As the title says I got error lines when generating the documentation with ProbeJS for my [Modpack](https://github.com/Lauriichan/ABuildersLife/) ([Modrinth](https://modrinth.com/modpack/a-builders-life) - might not be available yet because of review).
The errors share following pattern:
`setSprites(<set-?>: Internal.TextureAtlasSprite_[]): void;`
Every error has the `<set-?>` part as parameter name.

If you need a log even though I couldn't see any issues inside of it let me know.

**Comment by Prunoideae** [comment]
Illegal arg names my beloved... 

Will fix once I've done with my current work.

**Comment by lapraswastaken** [comment]
Did this end up getting fixed? I'm running into the same thing, using 1.20.1. Downgraded ProbeJS from 7.0.0 to 6.0.0 per #102 and I see `<set-?>` in place of various single-argument names which makes the linter freak out for the rest of the file.

For a workaround: I did a find/replace in VSCode to turn all `<set-?>`s into `arg` which solved that issue and is simple to redo after every /probejs dump.

Now I'm finding two instances of there being a `SHAPES_CACHE` object where all of the keys are integers, which the linter doesn't like.

![image](https://github.com/user-attachments/assets/9c9bef6e-a652-43bb-be6a-9a38621c58a1)
![image](https://github.com/user-attachments/assets/ee6f6426-93a7-4112-9933-3232990455bf)

I believe they're both coming from the mod Diagonal Fences [(link)](https://modrinth.com/mod/diagonal-fences).

The linter is happy after I did the arg replacement and removed the `SHAPES_CACHE` lines.
